rules/making-a-great-done-video/rule.md

```diff
@@ -16,14 +18,25 @@ redirects:
 created: 2023-05-08T23:25:40.698Z
 guid: 3444ccb7-0f6f-4ac6-97bc-035c491e6ec9
 ---
-For a great 'Done Video' here are the key things to remember:
+Although it can be daunting, creating a awesome 'Done Video' doesn't have to be hard. Luckily, we have you covered with these key tips to make your 'Done Video' great!
 
 <!--endintro-->
 
-1. **Generic** - Keep your video appealing and easy to understand for a global audience.
+## Content
+
+1. **Generic** - Keep your video appealing and easy to understand for a global audience. Generally, the video should not directly discuss the specific tasks of a PBI, but rather address the feature or change as a whole.
```

Comment:
Making it more succinct

```suggestion
1. **Generic** - Keep your video appealing and easy to understand for a global audience. The video should not directly discuss the tasks of a PBI, but rather address new the feature or changes wholistically. 
```

Reply:
I've gone with a combination of our changes 🙂